internal/shim-sev/src/syscall.rs

```diff
@@ -40,7 +40,7 @@ struct X8664DoubleReturn {
 /// # Safety
 ///
 /// This function is not be called from rust.
-#[inline(never)]
+// #[inline(never)]
```

Comment:
Just remove this. It will never be accepted and will turn into a hard error.

Reply:
ok